features/attached_commands.feature

```diff
@@ -27,6 +27,19 @@ Feature: Command behaviour when attached to an UA subscription
             """
             Successfully refreshed your subscription.
             """
+        When I run `ls /var/log/ubuntu-advantage*` as non-root
+        Then I will see the following on stdout:
+            """
+            /var/log/ubuntu-advantage.log
+            /var/log/ubuntu-advantage-timer.log
+            """
```

Comment:
@blackboxsw Looks like we're getting some legitimate CI failures. I think its that the `*` here isn't being expanded (we aren't using `shell=True` for our subprocess)

I think we can replace this section and the following with something like this to still use the globbing:
We might also need a `-1` to put each entry on its own line?
```suggestion
        When I run `sh -c "ls -1 /var/log/ubuntu-advantage*"` as non-root
        Then I will see the following on stdout:
            """
            /var/log/ubuntu-advantage.log
            /var/log/ubuntu-advantage-timer.log
            """
```